plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java

```diff
@@ -550,7 +551,12 @@ public HiveSessionProperties(
                         hiveConfig.getHudiCatalogName().orElse(null),
                         // Session-level redirections configuration does not work well with views, as view body is analyzed in context
                         // of a session with properties stripped off. Thus, this property is more of a test-only, or at most POC usefulness.
-                        true));
+                        true),
+                booleanProperty(
+                        LEGACY_ACID_MODIFICATION_ENABLED,
+                        "Enable performing legacy DELETE/UPDATE on Hive ACID transactional tables",
```

Comment:
As in config, remove or maye it scream out loud.

Reply:
Now throws

```
Query 20230118_094118_00005_fnwkj failed: DELETE on ACID transactional tables is not supported
when using the legacy UPDATE/DELETE implementation.Switch to the new implementation by
removing `deprecated.legacy-update-delete-implementation` system config propertyor setting
`legacy_update_delete_implementation` catalog session property to false.
```

Reply:
> As in config ...

BTW https://github.com/trinodb/trino/pull/15755 would make it simpler to define (config property, session property) pairs and keep them in sync with regards to description.

Reply:
Thank you for that PR.